subitem.go

```diff
@@ -21,21 +27,20 @@ type SubscriptionItemPriceDataParams struct {
 // For more details see https://stripe.com/docs/api#create_subscription_item and https://stripe.com/docs/api#update_subscription_item.
 type SubscriptionItemParams struct {
 	Params            `form:"*"`
-	ID                *string                                  `form:"-"` // Handled in URL
 	BillingThresholds *SubscriptionItemBillingThresholdsParams `form:"billing_thresholds"`
 	ClearUsage        *bool                                    `form:"clear_usage"`
+	OffSession        *bool                                    `form:"off_session"`
 	PaymentBehavior   *string                                  `form:"payment_behavior"`
 	Plan              *string                                  `form:"plan"`
 	Price             *string                                  `form:"price"`
 	PriceData         *SubscriptionItemPriceDataParams         `form:"price_data"`
-	ProrationDate     *int64                                   `form:"proration_date"`
 	ProrationBehavior *string                                  `form:"proration_behavior"`
+	ProrationDate     *int64                                   `form:"proration_date"`
 	Quantity          *int64                                   `form:"quantity"`
 	Subscription      *string                                  `form:"subscription"`
 	TaxRates          []*string                                `form:"tax_rates"`
 
-	// The following parameters are only supported on updates
-	OffSession *bool `form:"off_session"`
+	ID *string `form:"-"` // Deprecated
```

Comment:
As far as I can tell, this is completely redundant. It's not used by `subitem/client.go`, and since `form:"-"` it is not transmitted. Planning to remove in the next major.